src/components/CodeBlock.tsx

```diff
@@ -23,7 +23,7 @@ const StyledCopyIconBox = styled(View)`
 const StyledPre = styled.pre`
   text-align: left;
   padding: 5px;
-  overflow: scroll;
+  overflow-x: scroll;

```

Comment:
I think the y-axis scroll is more suitable for other pages since we use y-axis scroll in general. What do you think about it? If we want to put an x-axis scroll for code block on this page, I think it would be better to change it in Keccak FromString.tsx file.

Reply:
코드블록 컴포넌트의 높이가 코드 줄 수 에 맞춰서 늘어나기때문에, 세로로 스크롤이 필요가 없어요.